test/models/chan.js

```diff
@@ -35,7 +36,7 @@ describe("Chan", function() {
 				new User({name: "xPaw", mode: "~"}),
 				new User({name: "Max-P", mode: "@"}),
 			]});
-			chan.sortUsers();
+			chan.sortUsers({network: {options: {PREFIX: ["+", "%", "@", "&", "~"]}}});
```

Comment:
That's not how PREFIX is formatted by the way, which is why the test is failing. See https://github.com/kiwiirc/irc-framework/blob/48bb805653872e5f21210474fdfe439892be91b0/src/networkinfo.js#L12